core/controllers/AlbumController.js

```diff
@@ -0,0 +1,16 @@
+import Album from '../models/Album.js';
+
+class AlbumController {
+  async getAlbumsByAuthor(req, res, next) {}
+  async getAlbumsPublicByAuthor(req, res, next) {}
+  async getLikesById(req, res, next) {}
+  async getCountByAuthor(req, res, next) {}
```

Comment:
The AlbumSchema already has `count` attr, should we have this route?

Reply:
router.get('/count/:user-id/', photoController.getCountByAuthor); Dùng để hiển thị tổng số lượng album.